woocommerce-payments.php

```diff
@@ -91,6 +91,36 @@ function wcpay_jetpack_init() {
 			'priority'      => 5,
 		]
 	);
+
+	// When only WooCommerce Payments is active, minimize the data to send back to WPcom for supporting Woo Mobile apps.
+	$jetpack_config->ensure(
+		'sync',
+		array_merge_recursive(
+			\Automattic\Jetpack\Sync\Data_Settings::MUST_SYNC_DATA_SETTINGS,
+			[
+				'jetpack_sync_modules'           =>
+					[
+						'Automattic\\Jetpack\\Sync\\Modules\\Options',
+						'Automattic\\Jetpack\\Sync\\Modules\\Full_Sync',
+					],
+				'jetpack_sync_options_whitelist' => [ 'active_plugins' ],
```

Comment:
@hichamboushaba - I am thinking of syncing two options `blogdescription`, `blogname` options too in case that Woo Mobile apps want to use these info?

Reply:
@htdat yes, we need those, if they are not synced as part of the `MUST_SYNC_DATA_SETTINGS`, then please add them too 🙏

Reply:
Added here d85b1f1daa9d096a18482f4122cd19146dfa12b5 with some explanations p1644417342326889/1644409870.333429-slack-C02HHDEUEH2